lib/index.d.ts

```diff
@@ -1343,6 +1343,7 @@ declare namespace Stardog {
          *
          * @param {Connection} conn the Stardog server connection
          * @param {string} name the data source name
+         * @param {object} params the query string parameters
          */
         function online(conn: Connection, name: string): Promise<HTTP.Body>;
```

Comment:
it looks like we're still missing the params param here as well as the readme update

Reply:
looks like I added it to the wrong method too.. oops